docs/zh/using_starrocks/Cost_based_optimizer.md

```diff
@@ -102,11 +102,13 @@ StarRocks 提供灵活的信息采集方式，您可以根据业务场景选择
 
 触发自动采集的判断条件：
 
-- 上次统计信息采集之后，该表是否发生过数据变更。
+- 最新一次统计信息采集任务之后，该表是否发生过数据变更。
 - 分区数据是否发生过修改，未发生过修改的分区不做重新采集。
 - 采集是否落在配置的自动采集时间段内（默认为全天采集，可进行修改）。
 - 该表的统计信息健康度（`statistic_auto_collect_ratio`）是否低于配置阈值。
 
+  *需要注意的是如果某张表的数据变更后，手动触发了对它的抽样采集任务会使得条件一生效，不会触发这一轮的自动收集任务*
```

Comment:
What's the condition one?  What's the meaning of one round?

Reply:
means the first branch condition above